docs/src/pages/[platform]/index.page.tsx

```diff
@@ -146,313 +146,315 @@ const HomePage = ({ colorMode, setThemeOverride, themeOverride }) => {
   };
   const frameworkInstallScript = installScripts[platform.toString()];
   return (
-    <>
-      <View as="section" className="container">
-        <h1 className="docs-home-logo">
-          <HomeLogo />
-        </h1>
+    <View data-amplify-theme-override={themeOverride}>
+      <AmplifyProvider theme={theme} colorMode={colorMode}>
+        <View as="section" className="container">
```

Comment:
There is actually very little that has changed in this file (despite this compare not quite matching it up well).  The only changes to this page are importing the theme, moving themeOverride state to this component, and wrapping the return in a View and AmplifyProvider.